futures-local.ipynb

```diff
@@ -109,17 +115,30 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "6e1b7d49-92ba-4585-aa66-a8da5e19b166",
+   "id": "73e02d54-5231-48df-ad47-2d67c8532842",
    "metadata": {},
    "outputs": [],
-   "source": []
+   "source": [
+    "%%time\n",
+    "\n",
+    "results = []\n",
+    "for x in range(10):\n",
+    "    result = client.submit(inc, x)\n",
+    "    result = client.submit(dec, result)\n",
```

Comment:
Not a big deal either way. Just a thought that this might make things slightly more clear for readers. 

```suggestion
    "    result_inc = client.submit(inc, x)\n",
    "    result = client.submit(dec, result_inc)\n",
```

Reply:
I agree that that's more explicit, but I think it also calls attention to a part of this that I don't want people focused on.  The main thing to highlight here is the `client.submit` calls.  I want everything else here to be as uninteresting as possible.